tests/test_deprecations.py

```diff
@@ -1,11 +1,15 @@
 """Deprecation tests."""
 
 import json
+from unittest.mock import Mock
 
 import pytest
 
 from baybe import BayBE, Campaign
 from baybe.exceptions import DeprecationError
+from baybe.objective import Objective
+from baybe.objectives.base import Objective as NewObjective
```

Comment:
I would prefer to change the logic and have the "new" objective being called objective and the "old" objective being called "Dep_Objective" or similar.

Reply:
+1 for something like `NewObjective` and `OldObjective` in this file for clarity

Reply:
done